src/main/java/ru/yandex/qatools/ashot/shootingstrategy/ViewportPastingStrategy.java

```diff
@@ -8,29 +8,27 @@
 import java.awt.image.BufferedImage;
 import java.util.Set;
 
+import static ru.yandex.qatools.ashot.util.InnerScript.*;
+
 /**
  * @author <a href="pazone@yandex-team.ru">Pavel Zorin</a>
- *         <p/>
- *         Pastes together parts of screenshots
- *         Used when driver shoots viewport only
  */
 
-public abstract class VerticalPastingShootingStrategy extends HeadCuttingShootingStrategy {
+public class ViewportPastingStrategy extends ShootingStrategy<ViewportPastingStrategy> {
```

Comment:
Actually now it is not fully a strategy. It is some mix of wrapper and strategy. This class should be named properly